examples/slave/nuttx/xmc4800/main.cc

```diff
@@ -0,0 +1,65 @@
+#include <cstdint>
+
+#include "EmulatedEEPROM.h"
+#include "kickcat/Time.h"
+#include "kickcat/ESC/XMC4800.h"
+
+
+int main(int , char *[])
+{
+    using namespace kickcat;
+    printf("\n\n\n\n XMC hello \n");
+
+    kickcat::XMC4800 esc;
+    kickcat::EmulatedEEPROM eeprom;
+
+    esc.init();
+    eeprom.init();
+    uint16_t al_status;
+    uint16_t al_control;
+    esc.read(kickcat::reg::AL_STATUS, &al_status, sizeof(al_status));
+    esc.read(kickcat::reg::AL_CONTROL, &al_control, sizeof(al_control));
+    printf("al_status %x al_control %x  \n", al_status, al_control);
+
+
+    constexpr uint32_t pdo_size = 32;
+    uint8_t buffer_in[pdo_size];
+    uint8_t buffer_out[pdo_size];
+
+    // init values
+    for (uint32_t i=0; i < pdo_size; ++i)
+    {
+        buffer_in[i] = 3*i;
+        buffer_out[i] = 0xFF;
+    }
+
+    kickcat::SyncManagerConfig process_data_out = kickcat::SYNC_MANAGER_PI_OUT(0, 0x1000, pdo_size); // Process data out (master view), address consistent with eeprom conf.
```

Comment:
why using namespace kickcat if using it everywhere?

Reply:
Different timeline, I'll fix the repetition